src/slack.coffee

```diff
@@ -174,7 +174,10 @@ class SlackBot extends Adapter
       @receive new SlackTextMessage user, text, rawText, msg
 
   reactionAdded: (msg) =>
+    return if msg.user == @self.id
```

Comment:
I don't really understand what this line is doing...maybe worth an inline comment?